docs/publishing/advanced_publishing_options.md

```diff
@@ -108,16 +110,34 @@
     
     ![Calendar widget toolbar](img/selection_action_bar.png "Calendar widget toolbar")
     
-    #### Rescheduling or cancelling events
+    ## Rescheduling or cancelling publications
     
-    In case of publishable events (e.g. articles), you can change or cancel their planned publication by using the toolbar buttons.
+    In case of publishable Content items (e.g. articles), you can change or cancel their planned publication using the **Reschedule** or **Unschedule** buttons.
+    These buttons are accessible in the [Dashboard](#rescheduling-or-cancelling-in-dashboard) and the [Calendar widget](#rescheduling-or-cancelling-with-calendar-toolbar).
```

Comment:
those topics are just few lines below, do we need links to them?

Reply:
I may have overdone it. ;)